symengine/logic.cpp

```diff
@@ -352,6 +404,43 @@ RCP<const Boolean> logical_not(const RCP<const Boolean> &s)
     }
 }
 
+RCP<const Boolean> logical_xor(const set_boolean &s)
```

Comment:
From `x` I meant the same boolean. for example, `y < 0, y < 0` where `y` is a symbol.